sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -891,4 +894,19 @@ export class ServiceBusAtomManagementClient extends ServiceClient {
     const ruleResponse: RuleResponse = Object.assign(rule || {}, { _response: response });
     return ruleResponse;
   }
+
+  private buildResponse(fn: Function, response: HttpOperationResponse): any {
```

Comment:
a more precise type for `fn` would be appreciated.

Reply:
Since the function differs across operations, this might not be possible - any specific example of what you might have meant?

Reply:
Just going by usage, it looks like `(response) => void` would be appropriate?

Reply:
Updated